### PR TITLE
Minimal php installer

### PR DESCRIPTION
**Pull request by bobturneruk**
I've been trying to make a minimal installer following comments on pr #1. It seems to install, `php-fpm` is running, but I get `File not found.` when visiting the web site. Google suggests that I look at `nginx` settings, but I can't see how these should need to change. @willfurnass - can you spot anything obvious, here? Otherwise, I'll keep looking at it.

### CODE REVIEW

**Comment by bobturneruk** [comment]
This seems to work post commit 436a6083de8f8a623c191894ebad8a7bddc68941

File: defaults/main.yml [hunk 1]
```diff
@@ -1,18 +1,14 @@
 ---
-# Pass in a comma-separated list of repos to use (e.g. "remi,epel"). Used only
-# for RHEL/CentOS.
-php_enablerepo: ""
+# Pass in a comma-separated list of repos to use (e.g. "remi,epel"). 
+php_enablerepo: "remi-php73"
+remi_repo_url: "http://rpms.remirepo.net/enterprise/remi-release-{{ ansible_distribution_major_version }}.rpm"
+remi_repo_gpg_key_url: "http://rpms.remirepo.net/RPM-GPG-KEY-remi"
```
Comment (willfurnass) [inline defaults/main.yml]: Use HTTPS in literal URLs?

**Comment by willfurnass** [comment]
@bobturneruk Think this looks good.

**Review by willfurnass — APPROVED** [review]

**Comment by bobturneruk** [comment]
Thanks!

**Pull request merged**